build/gulpfile.scan.js

```diff
@@ -76,7 +76,14 @@ function nodeModules(destinationExe, destinationPdb, platform) {
 
 	const exe = () => {
 		return gulp.src(dependenciesSrc, { base: '.', dot: true })
-			.pipe(filter(['**/*.node', '!**/prebuilds/**/*.node']))
+			.pipe(filter([
+				'**/*.node',
+				// Exclude these paths.
+				// This one doesn't include symbols so BinSkim can't run
```

Comment:
More accurately, we don't build the prebuilt node files so we don't scan them.

Reply:
```suggestion
				// We don't build the prebuilt node files so we don't scan them
```